tests/test_solver.py

```diff
@@ -128,3 +129,22 @@ def test_solve_multiple():
     target_obj_pos = np.array([-0.1, -0.7, 0.3])
     sol = kinsol.solve_multiple(q_init, polygons, target_obj_pos)
     np.testing.assert_equal(polygon3, sol.target_polygon)
+
+def test_solve_multiple_with_realistic_data():
+    # Test using real polygon obtained from PR2 in the kitchen
+    config_path = "./config/pr2_conf.yaml"
+    config = SolverConfig.from_config_path(config_path, use_base=True)
+    kinsol = KinematicSolver(config)
+
+    polygons = get_sample_real_polygons()
+
+    q_init = np.ones(7) * 0.3
+
+    target_obj_pos = np.array([-0.1, 0.7, 0.3])
+    d_hover = 0.0
```

Comment:
In my environment, I use d_hover=0.05, but `d_hover=0.0` also reproduces my problem.